pegasus/sites.v3/hourofcode.com/views/resources/resources_page_nav.haml

```diff
@@ -10,6 +10,6 @@
 
 %nav.section-nav
   - sections.each do |section|
-    %a.link-button{href: "#" + section[:id]}
+    %a.link-button{href: section[:id] == "emails" ? "#sample-emails" : "#" + section[:id]}
```

Comment:
This is part of matching the old https://hourofcode.com/promote/resources#sample-emails skip link too.